pkg/minikube/node/start.go

```diff
@@ -922,6 +1034,52 @@ func prepareNone() {
 	}
 }
 
+func configureFlannelCNI() error {
+	err := cmd("kubectl apply -f https://raw.githubusercontent.com/vrapolinario/MinikubeWindowsContainers/main/kube-flannel.yaml")
```

Comment:
We need to update the version of flannel we are installing, and figure out how to remove the dependency on code that can change outside of our control

Reply:
creating an issue to track these for now